os/Makefile.unix

```diff
@@ -540,6 +540,9 @@ ifeq (download,$(firstword $(MAKECMDGOALS)))
 endif
 
 download:
+ifeq ($(CONFIG_KERNEL_CALLSTACK_WITH_SYMBOL),y)
```

Comment:
This is not a download procedure. This is not good. Let's move this to pass2 like making binary or make a new tab like post.